ansible/configs/kni-osp/default_vars_azure.yml

```diff
@@ -0,0 +1,14 @@
+---
+################################################################################
+################################################################################
+### Azure Specific Variables
```

Comment:
If this is an OSP specific config (`kni-osp`), is there a reason we have default vars for azure & ec2 as well?

Reply:
no, removing those

Reply:
@stencell can you resolve if this is ok now?